{
	"candidate": "CVE-2024-1066",
	"description": "An issue has been discovered in GitLab EE affecting all versions from 13.3.0 prior to 16.6.7, 16.7 prior to 16.7.5, and 16.8 prior to 16.8.2 which allows an attacker to do a resource exhaustion using GraphQL `vulnerabilitiesCountByDay`",
	"priority": "medium",
	"cvss": {
		"nvd": " CVSS:3.1/AV:N/AC:L/PR:L/UI:N/S:U/C:N/I:N/A:H [6.5 MEDIUM]"
	},
	"references": [
		"https://about.gitlab.com/releases/2024/02/07/security-release-gitlab-16-8-2-released/#resource-exhaustion-using-graphql-vulnerabilitiescountbyday",
		"https://gitlab.com/gitlab-org/gitlab/-/issues/420341",
		"https://www.cve.org/CVERecord?id=CVE-2024-1066"
	],
	"public_date": "2024-02-07 22:15:00 UTC",
	"packages": {
		"bionic": {
			"gitlab": {
				"name": "gitlab",
				"status": "DNE"
			}
		},
		"devel": {
			"gitlab": {
				"name": "gitlab",
				"status": "DNE"
			}
		},
		"esm-apps/xenial": {
			"gitlab": {
				"name": "gitlab",
				"status": "needs-triage"
			}
		},
		"focal": {
			"gitlab": {
				"name": "gitlab",
				"status": "DNE"
			}
		},
		"jammy": {
			"gitlab": {
				"name": "gitlab",
				"status": "DNE"
			}
		},
		"mantic": {
			"gitlab": {
				"name": "gitlab",
				"status": "DNE"
			}
		},
		"trusty": {
			"gitlab": {
				"name": "gitlab",
				"status": "DNE"
			}
		},
		"upstream": {
			"gitlab": {
				"name": "gitlab",
				"status": "needs-triage"
			}
		},
		"xenial": {
			"gitlab": {
				"name": "gitlab",
				"status": "ignored",
				"note": "end of standard support"
			}
		}
	}
}
